src/app/features/login/login/login.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { AuthService } from 'src/app/core/services/auth.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  public userEmail: string;
+  public userPassword: string;
+
+  constructor(private auth: AuthService) { }
```

Comment:
Just recommendation. Be better to naming service (or something importing into class) like full name with small first letter. For example
 constructor(private authService: AuthService) { }